R/radsl.R

```diff
@@ -89,11 +104,19 @@ radsl <- function(N = 400, # nolint
     AGEU = "YEARS",
     SEX = c("F", "M") %>% sample_fct(N, prob = c(female_prob, 1 - female_prob)),
     ARMCD = c("ARM A", "ARM B", "ARM C") %>% sample_fct(N),
-    RACE = c(
-      "ASIAN", "BLACK OR AFRICAN AMERICAN", "WHITE", "AMERICAN INDIAN OR ALASKA NATIVE",
-      "MULTIPLE", "NATIVE HAWAIIAN OR OTHER PACIFIC ISLANDER", "OTHER", "UNKNOWN"
-    ) %>%
-      sample_fct(N, prob = c(.55, .23, .16, .05, .004, .003, .002, .002)),
+
+    # if the user does not specify race values or probabilities
+    # i.e. by default race_list  = NULL and race_prob = NULL
+    # then RACE is computed using default values
+    # else RACE is computed using user-defined race_list  and race_prob
```

Comment:
This case can be broken by having set only one of the twos. Maybe you could add the defaults separately so it does not break